src/Razor/src/Microsoft.VisualStudio.RazorExtension/cgmanifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
-  "registrations": [
-    {
-      "component": {
-        "type": "git",
-        "git": {
-          "name": "microsoft/vscode",
-          "repositoryUrl": "https://github.com/microsoft/vscode",
-          "commitHash": "2e16c59fde65096c9ac2b3334f6b8b1fcbf87606"
-        }
-      },
-      "license": "MIT",
-      "description": "The files in the EmbeddedGrammars folder: csharp.tmLanguage, css.tmLanguage.json, html.tmLanguage.json and JavaScript.tmLanguage.json were pulled directly from VSCode. Also, the csharp-language-configuration.json, css-language-configuration.json, html-language-configuration.json and javascript-language-configuration.json files were pulled directly from VSCode.",
-      "version": "https://github.com/microsoft/vscode/tree/2e16c59fde65096c9ac2b3334f6b8b1fcbf87606"
-    }
-  ],
-  "version": 1
+    "$schema":  "https://json.schemastore.org/component-detection-manifest.json",
+    "registrations":  [
+                          {
+                              "component":  {
+                                                "type":  "git",
+                                                "git":  {
+                                                            "name":  "microsoft/vscode",
+                                                            "repositoryUrl":  "https://github.com/microsoft/vscode",
+                                                            "commitHash":  "ecb3421521db4764becfd198a79ca88b6aad73c2"
+                                                        }
+                                            },
+                              "license":  "MIT",
+                              "description":  "The files in the EmbeddedGrammars folder: csharp.tmLanguage, css.tmLanguage.json, html.tmLanguage.json and JavaScript.tmLanguage.json were pulled directly from VSCode. Also, the csharp-language-configuration.json, css-language-configuration.json, html-language-configuration.json and javascript-language-configuration.json files were pulled directly from VSCode.",
+                              "version":  "https://github.com/microsoft/vscode/tree/ecb3421521db4764becfd198a79ca88b6aad73c2"
+                          }
+                      ],
+    "version":  1
```

Comment:
Did you mean to change the formatting here so dramatically?

Reply:
Yes and no. No, in that Powershell wrote out the file like this, and I couldn't work out how to get it to stop, but yes, in that I don't think its a problem and at least future updates will be minimal.

Reply:
if it's `ConvertTo-Json` in powershell, I believe `-Compress` removes indentation/whitespace

Reply:
Yeah, but that makes it all one line, and even uglier IMO 😛

Reply:
ah, I haven't used it in a while. I tried 🤷